setupext/build_ext.py

```diff
@@ -315,8 +315,12 @@ def build_java_ext(self, ext):
             os.makedirs("build/classes", exist_ok=True)
             self.announce("  %s" % " ".join(cmd1), level=distutils.log.INFO)
             subprocess.check_call(cmd1)
+            manifest = None
             try:
                 for file in glob.iglob("native/java/**/*.*", recursive=True):
+                    if file.endswith("manifest.txt"):
+                        manifest = file
+                        continue
```

Comment:
This is another requirement for an agent.   We must add contents to the manifest and include it in the jar file.